File: CS50 - Week 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt user for the pyramide's height:
while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break

width = 1
space = height - 1

for _ in range(height):
    print(space * " ", end="")
    space = space - 1
    print(width * "#", end="")
    print("  ", end="")
    print(width * "#", end="")
    width = width + 1
    print()

File: CS50 - Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get positive integer from user.
    int n;
    do
    {
        n = get_int("Width: ");
    }
    while (n < 1 || n > 8);

    //Print out how wide is the wall
    for (int i = 0; i < n; i++) // genero un loop de n repeticiones
    {
        for (int k = n - i - 2; k >= 0; k--)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++) // genero un loop de i + 1 repeticiones de otra variable
        {
            printf("#"); // cada repetición va a generar un "#"
        }
        printf("\n"); // al final del loop, se imprime un salto de renglón.
    }
}